modules/installation-alibaba-permissions.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/installing-alibaba-account.adoc
+
+[id="installation-alibaba-permissions_{context}"]
+= Required Alibaba permissions for the RAM user
+
+
+To deploy all components of an {product-title}
+cluster, the Resource Access Management (RAM) user requires the following permissions:
+
+* ram:CreatePolicy
+* ram:GetPolicy
+* ram:CreatePolicyVersion
+* ram:DeletePolicy
+* ram:DetachPolicyFromUser
+* ram:ListPoliciesForUser
+* ram:AttachPolicyToUser
+* ram:CreateUser
+* ram:GetUser
+* ram:DeleteUser
+* ram:CreateAccessKey
+* ram:ListAccessKeys
+* ram:DeleteAccessKey
+
```

Comment:
@mtulio @jianli-wei @kwoodson @joelddiaz
Are these permissions correct for the RAM user that will run the installer?

Reply:
I can't imagine why the installer would be creating/deleting AccessKeys. And I see no such code in the installer repository, so my guess is that the answer to your question is 'no'.

Reply:
@joelddiaz 
I grabbed them from here:  [Cloud Credential Operator utility](https://github.com/DahuK/cloud-credential-operator/blob/de9cc738e2183f483819a062ce2c5175ea274e3b/docs/ccoctl.md#prerequisite)

Reply:
That list is the permissions for the `ccoctl` tool so that it can do some pre-installation setup. The installer likely makes all kinds of different/additional API calls.

Reply:
Removed from the assembly.